### PR TITLE
docs: tracks guide

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

Please add a detailed description of the change, weather it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

**Comment by dan-ziv** [comment]
Great guide

File: docs/tracks.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# Video, Audio and Text Tracks
+
+The Kaltura Player uses a comprehensive API to handle all kind of tracks, including video (bitrate), audio, and text (language) tracks.
+<br>This document shows how to use the API to implement and manage player tracks.
```
Comment (AnnaAleksandrowicz) [inline docs/tracks.md]: Is this ok in terms of my change?

Comment (yairans) [reply docs/tracks.md]: I'm not sure about 'implement'. the API is for managing tracks not for implementing.

**Comment by AnnaAleksandrowicz** [comment]
Ok so let’s use managing

Sent from my iPhone

On Jan 29, 2018, at 4:49 PM, Yair Ansbacher <notifications@github.com<mailto:notifications@github.com>> wrote:


@yairans commented on this pull request.

________________________________

In docs/tracks.md<https://github.com/kaltura/kaltura-player-js/pull/80#discussion_r164451749>:

> @@ -0,0 +1,174 @@
+# Video, Audio and Text Tracks
+
+The Kaltura Player uses a comprehensive API to handle all kind of tracks, including video (bitrate), audio, and text (language) tracks.
+<br>This document shows how to use the API to implement and manage player tracks.


I'm not sure about 'implement'. the API is for managing tracks not for implementing.

**Pull request merged**